other/materials_designer/create_interface_with_no_strain_matching.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<p>Prepend instead of Enrich, and it's not currently calculated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>